module1/build tools/core/src/main/java/com/epam/core/Utils.java

```diff
@@ -0,0 +1,16 @@
+package com.epam.core;
+
+import com.epam.task1.StringUtils;
+import java.util.Arrays;
+
+public class Utils {
+
+    public static void main(String[] args) {
+    }
+
+    public static boolean isAllPositiveNumbers(String... str) {
+        int before = str.length;
+        int after = (int) Arrays.stream(str).filter(StringUtils::isPositiveNumber).count();
+        return (before != 0 && before == after);
```

Comment:
Rows 13-14 can be simplified if you use allMatch terminal operation

Reply:
simplified method in last commit